dev/sparktestsupport/modules.py

```diff
@@ -431,6 +431,7 @@ def __hash__(self):
         "pyspark.ml.linalg.__init__",
         "pyspark.ml.recommendation",
         "pyspark.ml.regression",
+        "pyspark.ml.stat",
```

Comment:
Wait, do we need to update setup.py?  This is creating a module, not a package, right?